src/test/run-pass/binding/match-range.stderr

```diff
@@ -0,0 +1,20 @@
+warning: unreachable pattern
+  --> $DIR/match-range.rs:47:7
+   |
+LL |       _ => panic!("should match float range")
+   |       ^
+   |
+   = note: #[warn(unreachable_patterns)] on by default
```

Comment:
Same again; idk why this is unreachable (and it should cause a segfault when removed).